### PR TITLE
Fix organization select page styling issues

### LINKED ISSUE #355: Styling issues on organization select page
Three issues:

* Tooltip cut off on the right
* Tooltip cut off on the bottom
* Extra space at the beginning of the second row

screen shot 2015-11-04 at 11 43 26 am
screen shot 2015-11-04 at 11 43 20 am

### PR DESCRIPTION
**Pull request by beydogan**
This Fixes #355

### CODE REVIEW

**Comment by tarebyte** [comment]
I pulled down the branch and tried it out, looks great @beydogan!

@johndbritton it appears that the @beydogan has accepted the CLA (see https://github.com/education/classroom/pull/388) but for some reason it doesn't have that status on this PR.

This is what the new Classroom page looks like now

screen shot 2015-12-06 at 8 22 09 am

screen shot 2015-12-06 at 8 27 05 am

**Comment by beydogan** [comment]
I accepted CLA after I sent the PR, I think thats why. Should I send an
empty or..?

**Comment by tarebyte** [comment]
@beydogan @johndbritton might be able to restart the CLA for us since he's an :octocat: :grin:

**Comment by johndbritton** [comment]
@bkeepers @benbalter Here's a case where the CLA app isn't behaving as expected. Is there a way to trigger the build again? I suggest making the "Details" link, `https://cla.github.com/education/accept/beydogan`, have a way to refresh the status on the pull request.

File: app/views/organizations/_organization_select.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<%= button_tag type: 'submit', name: "organization[github_id]", class: 'btn organization-select-target js-navigation', value: org[:github_id] do %>
+<%= button_tag type: 'submit', name: "organization[github_id]", class: 'organization-select-target js-navigation', value: org[:github_id] do %>
   <%= image_tag "https://avatars.githubusercontent.com/u/#{org[:github_id]}?v=3&size=200", class: 'avatar organization-select-avatar', width: 100, height: 100, alt: "@#{org[:login]}" %>
   <span class="organization css-truncate css-truncate-target" title=<%= "@#{org[:login]}" %>><%= "@#{org[:login]}" %></span>
-<% end %>
+<% end %>
```
Comment (johndbritton) [inline app/views/organizations/_organization_select.html.erb]: Can you add a newline at the end of this file?

**Comment by johndbritton** [comment]
Looks great, thanks for the contribution @beydogan.

File: app/views/organizations/_organization_select.html.erb (hunk 1 above)
Comment (beydogan) [inline app/views/organizations/_organization_select.html.erb]: @johndbritton done :+1:

**Comment by tarebyte** [comment]
@beydogan can you merge the latest changes from master?

**Comment by tarebyte** [comment]
Thanks @beydogan!

**Pull request merged**